src/screens/Ticketing/Ticket/Details/DetailsScreen.tsx

```diff
@@ -83,8 +91,10 @@ export default function DetailsScreen({navigation, route}: Props) {
         {shouldShowValidTrainTicketNotice && (
           <MessageBox
             message={
-              isSingleTicket(firstTravelRight)
+              preassignedFareProduct?.type === 'single'
                 ? t(PurchaseOverviewTexts.samarbeidsbillettenInfo.single)
+                : preassignedFareProduct?.type === 'hour24'
+                ? t(PurchaseOverviewTexts.samarbeidsbillettenInfo.hour24)
                 : t(PurchaseOverviewTexts.samarbeidsbillettenInfo.period)
```

Comment:
I find these double "`x ? a : b`-statements" to be kind of hard to read. Could we extract this to a function instead? Something like

```ts
const getTrainTicketNoticeText = (fareProductType: string) => {
  if (fareProductType === 'single') return t(PurchaseOverviewTexts.samarbeidsbillettenInfo.single);
  if (fareProductType === 'hour24') return t(PurchaseOverviewTexts.samarbeidsbillettenInfo.hour24);
  return t(PurchaseOverviewTexts.samarbeidsbillettenInfo.period);
}
```

Reply:
I agree! I'll do that, thanks 👏